oshi-core/src/test/java/oshi/hardware/CentralProcessorTest.java

```diff
@@ -1,18 +1,18 @@
 /**
  * MIT License
- *
+ * <p>
```

Comment:
Our license header is written by a script which will, if all goes well, just overwrite your changes.  If all goes poorly, it might think it's missing and we end up with duplicate headers.  This doesn't go into the javadoc so we don't need the HTML tags here.

Reply:
I don't really understand how I changed it, maybe something Intellij did. what should I do?

Reply:
Just delete the added <p>?

Reply:
Yep, just put the header back to identically match all the other files.  Sometimes I even open other files to copy/paste!

Reply:
Cool, will fix. definitely something Intellij added by the way.